client/src/components/HealthLogToggle.js

```diff
@@ -16,7 +16,7 @@ export default function HealthLogToggle() {
         <Button onClick={setShowMeMore} variant={toggleValue === 'showMeMore' && "contained"}>Show me more</Button>
       </ButtonGroup>
       </Container>
-      {toggleValue === 'myHealthLog' && <SymptomsTracker />}
+      {/* {toggleValue === 'myHealthLog' && < />} */}
```

Comment:
Why are we changing this/commenting it out?

Reply:
Ohh I deleted it sorry! We don't need it there instead we need to put myHealthLog component with the calendar

Reply:
Ah okay, that makes sense!